### ISSUE TITLE
feature: Phylo interface to ete3

### ISSUE THREAD

**Issue by rwst** [biopython/biopython#1974]
In #1247 `Phylo.draw_graphviz` was deprecated. An interface to ete3 is suggested to provide a replacement: http://etetoolkit.org/docs/latest/index.html

Ete3 installs via pip and depends on PyQt5. That gave me some headache because my Linux system does not provide the needed dbus version. However, the GUI need not be used---image files can be generated as PNG or SVG and displayed with matplotlib. The interface to ete3 would be accessible using `Phylo.draw` and provide only basic functionality. For more users would get the underlying/converted ete3 tree object from a member function of `Bio.Phylo.Newick.Tree`.

**Comment by peterjc** [comment]
Thoughts @etal?

**Comment by etal** [comment]
I agree with removing `draw_graphviz` and interfacing with ETE instead.

**Comment by Darklod** [comment]
How about this for the conversion to ete3 in `Phylo.Newick`?

```python
from Bio.Phylo import BaseTree
from Bio import Phylo
from ete3 import Tree as EteTree
import tempfile

...

class Tree(BaseTree.Tree):

    ...

    def to_ete3(self):
        with tempfile.NamedTemporaryFile(mode="w") as tmp:
            Phylo.write(self.root, tmp, 'newick')
            tmp.flush()
            return EteTree(tmp.name)
```

**Comment by peterjc** [comment]
I'd want you to move the ``from ete3 import Tree as EteTree`` import into the ``to_ete3`` function (so that we're not adding a dependency for everyone - especially one with heavy dependencies of its own).

I'll defer to @etal, but to me this is better suited to an example in the documentation than code in Biopython.
